core/vtk/ttkMandatoryCriticalPoints/vtk.module

```diff
@@ -1,6 +1,4 @@
 NAME
  ttkMandatoryCriticalPoints
 DEPENDS
-  VTK::FiltersCore
-PRIVATE_DEPENDS
-  VTK::CommonCore
+ ttkAlgorithm
```

Comment:
@julien-tierny this change is what I am mentioning in #451 
@pierre-guillou do not worry, if this PR is merged I can rebase mine and fix this on my side

Reply:
Never-mind, we can keep the mechanism with `ttkAlgorithm`.